examples/buildpacks/buildpacks.cue

```diff
@@ -30,9 +30,6 @@ frsca: pipelineRun: "cache-image-pipelinerun-": spec: {
 	}, {
 		name:  "CACHE_IMAGE"
 		value: _CACHE_IMAGE
-	}, {
-		name: "ENV_VARS"
-		value: [""]
```

Comment:
@bradbeck just verifying this is intentional because we had just added it in #248

Reply:
I believe that's the workaround removed by this PR.

Reply:
@sudo-bmitch Yep, the updated buildpacks pipeline definition includes this same value as its default.